src/App.js

```diff
@@ -51,21 +88,23 @@ const App = () => {
   }
 
   const resetGame = () => {
-    // Complete in Wave 4
+
   }
+  // Complete in Wave 4
+
 
   return (
     <div className="App">
       <header className="App-header">
         <h1>React Tic Tac Toe</h1>
-        <h2>The winner is ... -- Fill in for wave 3 </h2>
```

Comment:
Here is a ternary operator you can have to show the current player if the winner is null `{winner === null ? `Current Player ${ currentPlayer }` : `Winner is ${ winner }`}`